microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -170,6 +164,8 @@ To complete registration for your customer, you must create a CSV file.
 - Device model (example: Surface Laptop)
 - Device serial number
 
+[//]: # (hang on--how come the values have to match exactly here, but it doesn't matter with the self-service process??)
+
```

Comment:
Nice catch - it is a bit odd, but accurate. The reason is that for customers we use the Hardware Hash to look up the device, so the other values aren't really needed. For PArtners we have no hash, and so we use the other three values instead. Since we're using them to do a look-up, they need to be **perfect**

Reply:
Ah so--OK, that'll be something I'll really emphasize (in both places). That way in case someone reads both they won't get confused or trip over the inconsistency (HOPEFULLY).

Reply:
Rgr, good plan. thanks man!